client/src/pages/dashboard.tsx

```diff
@@ -0,0 +1,80 @@
+import { format, fromUnixTime, getUnixTime } from 'date-fns';
+import { DashboardLayout } from '../components/DashboardLayout';
+import { Overview } from '../components/Overview';
+import { DashboardData, PriceResult } from '../types/FileData';
+import { Container } from '../ui/Container';
+
+interface DashboardProps {
+  statsData: DashboardData;
+}
+
+const Dashboard = ({ statsData }: DashboardProps) => {
+  return (
+    <Container
+      css={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '$5',
+        py: '$5',
+
+        '@lg': {
+          gap: '$20',
+          flexDirection: 'row',
+          py: '$10',
+          height: '100vh',
+        },
+
+        '@xl': {
+          gap: '128px',
+        },
+      }}
+    >
+      <DashboardLayout />
+      <Overview statsData={statsData} />
+    </Container>
+  );
+};
+
+export async function getStaticProps() {
+  const res = await fetch(process.env.DASHBOARD_API_URL!);
```

Comment:
In general, it's better to save the root API URL in the config and then use it in the code, so for example here you could do
```suggestion
  const res = await fetch(`${process.env.API_URL!}/dashboard`);
```